app/connectors/AmlsConnector.scala

```diff
@@ -46,12 +47,12 @@ class AmlsConnector @Inject()(val http: HttpClient,
     val postUrl = s"$url/$accountType/$accountId/$safeId"
     val prefix = "[AmlsConnector][subscribe]"
     // $COVERAGE-OFF$
-    Logger.debug(s"$prefix - Request Body: ${Json.toJson(subscriptionRequest)}")
+    logger.debug(s"$prefix - Request Body: ${Json.toJson(subscriptionRequest)}")
```

Comment:
not for this PR, but we can remove all the debug logs as these won't be printed anyway in live environment. but leave it for now